KnockKnock-iOS/Network/KKNetworkManager.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NetworkManager.swift
+//  KnockKnock-iOS
+//
+//  Created by Daye on 2022/06/07.
+//
+
+import Foundation
+
+import Alamofire
+
+class KKNetworkManager {
+  typealias Success<T> = ((T) -> Void)
+  typealias Failure = ((_ error: Error) -> Void)
+
+  static let shared = KKNetworkManager()
+
+  func request<T>(object: T.Type,
+                  router: KKRouter,
+                  success: @escaping Success<T>,
+                  failure: @escaping Failure) where T: Decodable {
```

Comment:
```suggestion
  func request<T>(
    object: T.Type,
    router: KKRouter,
    success: @escaping Success<T>,
    failure: @escaping Failure
  ) where T: Decodable {
```
저희 Swift 스타일 가이드상으론 이렇게 개행하는게 맞아보입니다 👍